### PR TITLE
Add solar irradiance and wind speed for the selected site

### PR DESCRIPTION
**Pull request by paulapreuss**
- Basic display of weather data on resources view
- Added functions in backend to get wind data and raw data (irradiance/wind speed instead of just electricity output)

This is just very basic for now to gather feedback about if we want to keep this information in some form.

### CODE REVIEW

File: app/templates/wefe/steps/choose_location.html [hunk 1]
```diff
@@ -54,8 +54,8 @@
 
         <script>
             /* Map Show and Latitude-Longitude data autofill */
-            var mymap = L.map('mapid').setView([8.786469, 7.433806], 6);
-						var marker;
+            var mymap = L.map('mapid').setView([49.45, 13.89], 4);
```
Comment (Bachibouzouk) [inline app/templates/wefe/steps/choose_location.html:57]: We could get the rough lat lon from the IP and center the map around those with https://docs.djangoproject.com/en/5.1/ref/contrib/gis/geoip2/

**Review by Bachibouzouk — APPROVED** [review]
Thanks a lot for implementing this @paulapreuss :) 

I have a few comments:
- The location chosen in step 1, does not show on the map with a marker
- When I change the location, the PV and wind timeseries are not updated accordingly (I think we need to either remove the ts objects created or add lat lon indexing. We will have access to all weather timeseries worldwide on our local DB

File: app/projects/services.py [hunk 2]
```diff
@@ -295,28 +295,60 @@ def get_pv_output(self, coordinates):
...
+        ##
 
-        pv_data = pd.read_json(json.dumps(parsed_response["data"]), orient="index")
-        metadata = parsed_response["metadata"]
+        url = self.api_base + "data/wind"
 
-        self.data = pv_data
+        args = {
+            "lat": 34.125,
+            "lon": 39.814,
```
Comment (Bachibouzouk) [inline app/projects/services.py]: This is hardcoded for wind?

**Review by Bachibouzouk — APPROVED** [review]

File: app/projects/services.py (hunk 2 above)
Comment (paulapreuss) [reply app/projects/services.py]: Ah, good catch. I copy/pasted from RN and forgot to update this.

**Comment by paulapreuss** [comment]
> Thanks a lot for implementing this @paulapreuss :)
> 
> I have a few comments:
> 
> * The location chosen in step 1, does not show on the map with a marker
> * When I change the location, the PV and wind timeseries are not updated accordingly (I think we need to either remove the ts objects created or add lat lon indexing. We will have access to all weather timeseries worldwide on our local DB

Will handle by deleting the objects for now if the location is changed. Long term if we are using different solution that the RN API I guess most of this functionality will be changed a bit anyway.

File: app/templates/wefe/steps/choose_location.html (hunk 1 above)
Comment (paulapreuss) [reply app/templates/wefe/steps/choose_location.html:57]: I created a new issue for this so we can do it as a nice to have later

**Pull request merged**